src/tests/end-to-end/common/page-controllers/html-report-page.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as Playwright from 'playwright';
+import { CommonSelectors } from 'tests/end-to-end/common/element-identifiers/common-selectors';
+import { Page, PageOptions } from './page';
+
+export class HtmlReportPage extends Page {
+    constructor(underlyingPage: Playwright.Page, options?: PageOptions) {
+        super(underlyingPage, options);
+    }
+
+    public async ensureNoModals(): Promise<void> {
```

Comment:
Is this a copy/paste leftover? I don't see this being used anywhere

Reply:
Good catch!